package.json

```diff
@@ -56,18 +55,17 @@
     "eslint-plugin-react-hooks": "4.6.0",
     "fs-extra": "11.1.1",
     "husky": "8.0.3",
-    "jest": "29.7.0",
-    "jest-environment-jsdom": "29.7.0",
+    "jsdom": "^22.1.0",
```

Comment:
I think the good approach is don't use '^' anyway, but here we have deps bot, and they are looking for new versions, as you can see in other packages we don't have it, but here @Jonghakseo could say more if he want this, let's do it :)